### PR TITLE
Improve documentation on secondary sources

### PR DESCRIPTION
**Pull request by hagenw**
This mentions the Gauss grid for spherical arrays and adds one example plot of a logarithmically spaced linear array. You can preview the rendered page at http://matlab.sfstoolbox.org/en/ssd_doc/secondary-sources/.

In addition, this pull request adds a small comment to the `logspread` entry in `SFS_config.m`

### CODE REVIEW

File: doc/secondary-sources.rst [hunk 1]
```diff
@@ -36,6 +36,31 @@ Linear array
 
     Linear loudspeaker array with a length of 3m consiting of 21 loudspeakers.
 
+You can also logarithmically space the secondary sources for a linear array
```
Comment (fietew) [inline doc/secondary-sources.rst]: maybe: You can also create logarithmic spacing for a linear array

Comment (hagenw) [reply doc/secondary-sources.rst]: Thanks, I changed this.

**Comment by hagenw** [comment]
Regarding sphinx, for me it compiles also locally. Try:
```bash
virtualenv --python="python3" --no-site-packages "tmp"
source tmp/bin/activate
pip install sphinx
pip install sphinx_rtd_theme
cd doc
sphinx-build -b html -d ./_build/doctrees . ./_build/html-preview/
```

**Comment by fietew** [comment]
Sry, misunderstanding. "I did not compile via sphinx"

**Comment by hagenw** [comment]
OK, then I will clean up the last two commits and rebase the changes.

**Pull request merged**